packages/compass-e2e-tests/tests/in-use-encryption.test.ts

```diff
@@ -244,12 +240,10 @@ describe('CSFLE / QE', function () {
       let compass: Compass;
       let browser: CompassBrowser;
       let plainMongo: MongoClient;
-      let hasRangeSupport = false;
 
       before(async function () {
         compass = await beforeTests();
         browser = compass.browser;
-        hasRangeSupport = serverSatisfies('>= 6.2.0', true);
```

Comment:
I removed this check now because 7+ implies 6.2+, but maybe this is also something to be tested as part of https://jira.mongodb.org/browse/COMPASS-6602?